discovery-common/src/test/java/com/nepxion/discovery/common/entity/InstanceEntityWrapperTest.java

```diff
@@ -0,0 +1,72 @@
+package com.nepxion.discovery.common.entity;
+
+import com.nepxion.discovery.common.constant.DiscoveryConstant;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import static com.nepxion.discovery.common.entity.InstanceEntityWrapper.*;
+
+public class InstanceEntityWrapperTest {
+
+    private InstanceEntity instanceEntity = new InstanceEntity();
+    private HashMap<String, String> hashMap = new HashMap<>();
```

Comment:
🚫 That makes the tests dependent.

Reply:
Removed the global variable and moved it into the methods